diagnostic.md

```diff
@@ -7,27 +7,36 @@ Write your answers inside this file, where it's indicated by the comments.
     repository in your answer.
 
     ```sh
-    <!-- Remove this comment and place your answer here. -->
+    <!--
+    git clone git@github.com:kenvng/git-basics-diagnostic.git
+    -->
     ```
 
 2.  What do you do after cloning a repository, but before starting work?
 
     ```sh
-    <!-- Remove this comment and place your answer here. -->
+    <!--
+    cd into the folder/directory with the file you need to work on and
+    enter: atom <filename with extension> to open it in atom.
```

Comment:
At least in the context of this class's diagnostics, before you open your editor and begin editing, you should create a new branch and switch to it.